AzureMonitorAgent/ama_tst/modules/errors.py

```diff
@@ -0,0 +1,103 @@
+import copy
+import subprocess
+
+from error_codes import *
+
+# backwards compatible input() function for Python 2 vs 3
+try:
+    input = raw_input
+except NameError:
+    pass
+
+# error info edited when error occurs
+error_info = []
+
+# list of all errors called when script ran
+err_summary = []
+
+
+
+# # set of all errors which are actually warnings
+warnings = set([WARN_FILE_PERMS, WARN_LOG_ERRS, WARN_LOG_WARNS, WARN_LARGE_FILES, \
```

Comment:
Same note regarding the warnings as above - you can keep the infrastructure, but I'd recommend removing any warnings you're not using